spec/did.md

```diff
@@ -0,0 +1,79 @@
+
+## DID Document Data Model
+
+Following from [this data model](https://www.w3.org/TR/did-core/#core-properties).
+
+| Property      | JSON Representation | Required | Notes          |
+| ------------- | ------------------- | -------- | -------------- |
+| `id`          | String              | Yes      | Must be a URI. |
+| `@context`    | Array of strings    | No       | Depends on the DID method. |
+| `controller`  | Array of strings    | No       | Depends on the DID method. Strings must be URIs. |
+| `alsoKnownAs`        | Array of strings | No   | Depends on the DID method. Strings must be URIs. |
+| `verificationMethod` | Array of [Verification Methods](#verification-method-data-model) | Yes | There must be at least one Verification Method in each DID Document. |
+| `authentication`     | Array of strings | No   | String values must be fully qualified DID URIs (e.g. `did:example:abcd#key-1` over `#key-1`). |
+| `assertionMethod`    | Array of strings | No   | String values must be fully qualified DID URIs (e.g. `did:example:abcd#key-1` over `#key-1`). |
+| `keyAgreement`       | Array of strings | No   | String values must be fully qualified DID URIs (e.g. `did:example:abcd#key-1` over `#key-1`). |
+| `capabilityInvocation` | Array of strings | No | String values must be fully qualified DID URIs (e.g. `did:example:abcd#key-1` over `#key-1`). |
+| `capabilityDelegation` | Array of strings | No | String values must be fully qualified DID URIs (e.g. `did:example:abcd#key-1` over `#key-1`). |
+| `service`            | Array of [Services](#service-data-model) | No | - |
+
+**Additional Notes:**
+- No [JSON-LD processing](https://www.w3.org/TR/did-core/#consumption-0) is performed.
+- Each [Verification Method](https://www.w3.org/TR/did-core/#verification-methods) must have at least one [Verification Relationship](https://www.w3.org/TR/did-core/#verification-relationships).
+
+### Verification Method Data Model
+
+Following form [this data model](https://www.w3.org/TR/did-core/#verification-methods).
+
+| Property      | JSON Representation | Required | Notes          |
+| ------------- | ------------------- | -------- | -------------- |
+| `id`          | String              | Yes      | Must be a fully qualified DID URI (e.g. `did:example:abcd#key-1`). |
+| `type`        | String              | Yes      | Must be a URI. |
+| `controller`  | String              | Yes      | Must be a URI. |
+| `publicKeyJwk`        | Object      | No       | Preferred. Represents a [JWK](https://www.w3.org/TR/did-core/#bib-rfc7517). Either this or `publicKeyMultibase` must be present. |
+| `publicKeyMultibase`  | String      | No       | Not preferred. Either this or `publicKeyJwk` must be present. |
```

Comment:
are we cool with dropping this or adding it in later on? will be tough to find time to implement in `dart`. can find time if we feel it's important

Reply:
To add my personal opinion to [the question](https://github.com/TBD54566975/web5-spec/pull/122#issuecomment-1981342822), I am comfortable dropping `publicKeyMultibase` as a requirement for the first iteration of this spec.

@decentralgabe if you know of any significant adoption of Multibase format for DIDs by US state DMVs or EU member states, that could be a reason to include.  However, given our heavy reliance on the JWK format in every SDK we've built to date and that DID JWK and DID DHT currently have no plans to support verification material in any format other than `publicKeyJwk`, it doesn't seem unreasonable to exclude.

Reply:
let's drop it